packages/strapi-plugin-graphql/hooks/graphql/index.js

```diff
@@ -113,12 +124,13 @@ module.exports = strapi => {
           return typeof formatError === 'function' ? formatError(err) : err;
         },
         validationRules: [depthLimit(strapi.plugins.graphql.config.depthLimit)],
-        tracing: _.get(strapi.plugins.graphql, 'config.tracing', false),
+        tracing: _.get(serverConfig, 'tracing', false),
```

Comment:
you can directly set tracing: false as it will be overwritten byt the serverConfig, some goes for introspection and engine :)

Reply:
Oops! You are totally right! Rush!